src/status_im/utils/keychain.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im.utils.keychain
+  (:require [cljs.core.async :as async]
+            [taoensso.timbre :as log]
+            [status-im.react-native.js-dependencies :as rn]))
+
+(def username "status-im.encryptionkey")
+
+(defn- random-key []
```

Comment:
> Math.random() does not provide cryptographically secure random numbers. Do not use them for anything related to security. Use the Web Crypto API instead, and more precisely the window.crypto.getRandomValues() method.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random

Reply:
@cammellos geth doesn't persist keys it will disappear after logout, but it makes me think maybe we could even use a child key that is derived from the master key so we don't have to store it in the keychain